packages/dashboard/app/components/settings/sections/__tests__/settings-default-descriptions.test.tsx

```
@@ -273,6 +273,9 @@ const SETTING_DESCRIPTION_KEYS: Record<string, string> = {
 
 /** Setting keys intentionally not surfaced as a plain Settings UI description field, with reasons. */
 const NOT_SURFACED_ALLOWLIST: Record<string, string> = {
+  // Internal migration/inbox bookkeeping — not plain Settings UI description fields.
+  sqliteMigrationNotice: "engine/migration bookkeeping; not a Settings description field",
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)